packages/marathon/build

```diff
@@ -23,11 +23,13 @@ EnvironmentFile=/opt/mesosphere/environment
 EnvironmentFile=/opt/mesosphere/etc/marathon
 EnvironmentFile=-/opt/mesosphere/etc/marathon-extras
 EnvironmentFile=-/var/lib/dcos/marathon/environment.ip.marathon
+EnvironmentFile=-/var/lib/dcos/marathon/optional.flags.marathon
 ExecStartPre=/bin/ping -c1 leader.mesos
 ExecStartPre=/opt/mesosphere/bin/bootstrap dcos-marathon
 ExecStartPre=/bin/bash -c 'echo "HOST_IP=\$(\$MESOS_IP_DISCOVERY_COMMAND)" > /var/lib/dcos/marathon/environment.ip.marathon'
 ExecStartPre=/bin/bash -c 'echo "MARATHON_HOSTNAME=\$(\$MESOS_IP_DISCOVERY_COMMAND)" >> /var/lib/dcos/marathon/environment.ip.marathon'
 ExecStartPre=/bin/bash -c 'echo "LIBPROCESS_IP=\$(\$MESOS_IP_DISCOVERY_COMMAND)" >> /var/lib/dcos/marathon/environment.ip.marathon'
+ExecStartPre=-/bin/bash -c '[ -n "\$MARATHON_DEFAULT_NETWORK_NAME" ] && echo OPTFLAG_DEFAULT_NETWORK_NAME=\\"--default_network_name \$MARATHON_DEFAULT_NETWORK_NAME\\" > /var/lib/dcos/marathon/optional.flags.marathon'
```

Comment:
this violates https://github.com/dcos/dcos/blob/master/docs/systemd-rules.md ("All Exec and ExecStart lines must be trivial")